charts/cronjob/templates/cronjob.yaml

```diff
@@ -9,6 +9,9 @@ metadata:
     heritage: {{ .Release.Service }}
 spec:
   schedule: {{ .Values.schedule | quote }}
+  {{- if eq .Values.isOnDemand true}}
```

Comment:
Should be enough: `{{- if .Values. isOnDemand }}`. Check other if we have in this file.

Reply:
Hello @lojzatran @butenkor - please check my update here 3c5f47e